src/style.css

```
@@ -0,0 +1,103 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 15px 14px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+ 
+  --blue-button-color: #00acdc;
+}
+
+.product-card__image {
+  display: flex;
+  margin: 14px 5px 45px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__name {
+  margin-bottom: 6px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.product-card__product-code {
+  margin-bottom: 15px;
+}
+
+.product-card__product-code,
+.product-card__response,
+.product-card__label {
+  font-size: 11px;
+  font-weight: 400;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+}
+
+.stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
```

These styles should apply only for corresponding modifier `stars--4`

```suggestion
stars--4 > .stars__star:nth-child(-n + 4) {
  background-image: url(./images/star-active.svg);
}
```